meta/runtime.yml

```diff
@@ -13,6 +13,11 @@ action_groups:
   - logs_query_definition
   - logs_resource_policy
   - rds_db_proxy
+  - memorydb_acl
```

Comment:
I'm not sure what's going on, but since the memorydb* modules have been already added to meta/runtime.yml https://github.com/ansible-collections/amazon.cloud/blob/main/meta/runtime.yml#L65 I'm not understanding why I don't see this change.